source/index.rst

```diff
@@ -48,5 +52,6 @@ usually written in bash, but other languages are supported as well.
 
    self
    getting_started
+   tutorial
```

Comment:
This shows up on the left hand side for all pages except "contributing" -- see https://portal.nersc.gov/project/e3sm/forsyth/zppy_docs_2021_06_07/html/contributing.html. I don't know why that would be the case.

Reply:
Will be fixed in #77.